package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ngrok-doc",
   "version": "0.0.0",
-  "private": true,
+  "private": false,
```

Comment:
Is this required for algolia? This property prevents npm publishing.

Reply:
```suggestion
  "private": true,
```